src-vue/src/components/LanguageSelector.vue

```diff
@@ -46,6 +46,10 @@ export default defineComponent({
                 value: 'it',
                 label: 'Italiano'
             },
+            {
+                value: 'zh_Hans',
+                label: '简体中文'
```

Comment:
I don't speak Chinese so I used Google Translate for that key. Maybe a native speaker like @zxcPandora or @wolf109909 could confirm whether it's correct ^^"

Reply:
Seems to be fine: https://en.wiktionary.org/wiki/%E7%AE%80%E4%BD%93%E4%B8%AD%E6%96%87